src/Popper.jsx

```diff
@@ -139,13 +139,15 @@ class Popper extends Component {
       left,
       position,
     } = this.state.data.offsets.popper
+    const styles = this.state.data.styles
 
     return {
       position,
       top:        0,
       left:       0,
       transform:  `translate3d(${left}px, ${top}px, 0px)`,
       willChange: 'transform',
+      ...styles,
```

Comment:
@FezVrasta so are you saying the transform values and such do not need to be added since `styles` is used now?